test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -968,13 +1072,19 @@ private String firstTimeStampFor(int initialTZOffset, int requestTZOffset, Sched
         return activities.stream().map(ScheduledActivity::getGuid).collect(Collectors.toSet());
     }
     
+    // TODO: Make endsOn a constant, no reason for it to be here.
     private ScheduleContext.Builder createScheduleContext(DateTime endsOn) {
         Map<String,DateTime> events = Maps.newHashMap();
-        events.put("enrollment", ENROLLMENT);
+        events.put("enrollment", ENROLLMENT.withZone(DateTimeZone.UTC));
         
-        return new ScheduleContext.Builder().withStudyIdentifier(TEST_STUDY).withInitialTimeZone(DateTimeZone.UTC)
-                .withStartsOn(NOW).withAccountCreatedOn(ENROLLMENT.minusHours(2)).withEndsOn(endsOn)
-                .withHealthCode(HEALTH_CODE).withUserId(USER_ID).withEvents(events);
+        return new ScheduleContext.Builder().withStudyIdentifier(TEST_STUDY)
+                .withInitialTimeZone(DateTimeZone.forOffsetHours(-7))
+                .withAccountCreatedOn(ENROLLMENT.minusHours(2))
+                .withStartsOn(STARTS_ON)
+                .withEndsOn(endsOn)
+                .withHealthCode(HEALTH_CODE)
+                .withUserId(USER_ID)
+                .withEvents(events);
```

Comment:
Why are we changing this? We specifically set up the tests this way to test the time zone fix. If we change the schedule context here, we'll need to reason through N different test cases to make sure we didn't remove a test case or accidentally break something.

I'd strongly recommend keeping this intact. If tests break because of it, it's very likely that we accidentally introduced a bug, which we need to understand.